modules/executive/components/ExecutiveOverviewCard.tsx

```diff
@@ -81,17 +81,17 @@ export default function ExecutiveOverviewCard({
         <Flex sx={{ justifyContent: 'space-between' }}>
           <Box>
             <Flex sx={{ flexDirection: 'column', height: '200px' }}>
-              <Link
-                href={{ pathname: '/executive/[proposal-id]' }}
-                as={{ pathname: `/executive/${proposal.key}` }}
-                passHref
+              <InternalLink
+                href={'/executive/[proposal-id]'}
+                title="View executive details"
+                as={`/executive/${proposal.key}`}
```

Comment:
Nice, great idea. One note, I believe the `as` property is no longer required and we can pass the dynamic route straight into `href`. Check this out for more: https://nextjs.org/docs/api-reference/next/link

Reply:
yeah

Reply:
> Nice, great idea. One note, I believe the `as` property is no longer required and we can pass the dynamic route straight into `href`. Check this out for more: https://nextjs.org/docs/api-reference/next/link

Sweet, thanks for the heads up. Nice thing is in the future if a change like that occurs again, we'll hopefully only have to update it in one place.